app/services/search.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+angular.module("app")
+
+.factory('search', ['$http', '$q', 'NORDMART_CONFIG', 'Auth', '$location', function($http, $q, NORDMART_CONFIG, $auth, $location) {
+	var factory = {}, products, baseUrl;
+
+	if ($location.protocol() === 'https') {
+		baseUrl = (NORDMART_CONFIG.SECURE_API_ENDPOINT.startsWith("https://")
```

Comment:
I can't see any PR for https://github.com/stakater-lab/stakater-nordmart-gateway
`Gateway` is the only exposed service and all communication from client-side is supposed to be redirected from gateway to other micro-services example, search. 
You'll need to add a handler for `/api/v1/product-search` in there as well

Reply:
Here it is: https://github.com/stakater-lab/stakater-nordmart-gateway/pull/25